api/v3/openapi.yaml

```diff
@@ -1610,6 +1612,8 @@ components:
                   - SET
                   - DELETE
                 nullable: true
+        validate_only:
```

Comment:
if I understand correctly, the default value for validate_only if not specified in the request is false, shall we also comment on that as well?

Reply:
Correct. I'm not sure if there's anything better than explicitly calling out that the dry-run mode requires specifying that property and setting it to `true`. I've added this to the descriptions of both endpoints.